dingo_test.go

```diff
@@ -313,3 +313,30 @@ func TestInjector_InitModules(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Error(t, injector.InitModules(new(testInjectInvalid)))
 }
+
+type TestInjectStructRecInterface interface {
+	TestXyz()
+}
+
+type testInjectStructRecStruct struct {
+	test string
+}
+
+func (t testInjectStructRecStruct) TestXyz() {
+
+}
+
+func (t testInjectStructRecStruct) Inject(test string) {
+	t.test = test
+}
+
+func TestInjectStructRec(t *testing.T) {
```

Comment:
Can you add `t.Parallel()`? Then the tests can run a little faster and the linter will be happy.

Reply:
Done.